.travis.yml

```diff
@@ -0,0 +1,28 @@
+sudo: false
+language: android
+jdk: oraclejdk8
+
+env:
+  matrix:
+    - ANDROID_TARGET=android-24
+
+android:
+  components:
+    - platform-tools
+    - tools
+    - android-24
+    - build-tools-24.0.1
+    - extra-android-support
+    - extra-android-m2repository
+
+licenses:
+  - 'android-sdk-license-.+'
+
+before_install:
+  - echo yes | android update sdk --filter extra-android-m2repository --no-ui --force > /dev/null
+
+script:
+  - ./gradlew build test
```

Comment:
I suggest using `check` which includes `build` and `test` but also includes plugin outputs like jacoco (for code coverage).